manifests/01_config.yaml

```diff
@@ -18,6 +18,20 @@ data:
       blockers:
         name: openshift-group-b-blockers
         sharerID: 290313
+      staleClose:
+        name: openshift-group-b-lifecycle-close
+        sharerID: 290313
+        action:
+          setState: CLOSED
+          setResolution: WONTFIX
+          priorityTransitions:
+            - from: medium
+              to: low
```

Comment:
does this mean they go down the priority ladder first and then get closed?

Reply:
we might want to parametrize the 7d

Reply:
there should be no "high" in the list, because the stale controller degrade them beforehand... So only unspecified and medium.

7d: we will need to get rid of saved search then, because that is set there...